cranelift/fuzzgen/Cargo.toml

```diff
@@ -12,6 +12,8 @@ publish = false
 
 [dependencies]
 cranelift = { path = "../umbrella", version = "0.87.0" }
+cranelift-native = { path = "../native", version = "0.87.0" }
+cranelift-entity = { path = "../entity", version = "0.87.0" }
```

Comment:
Thanks for fixing the use of `ValueList`! I think you don't need a dependency on `cranelift-entity` now, right?

Reply:
Right, Fixed!